actionview/lib/action_view/helpers/translation_helper.rb

```diff
@@ -57,6 +57,17 @@ module TranslationHelper
       # that include HTML tags so that you know what kind of output to expect
       # when you call translate in a template and translators know which keys
       # they can provide HTML values for.
+      #
+      # Finally, to access the translated text along with the fully resolved
```

Comment:
The existing documentation as organized:

> [...] also performs three additional functions. [...] First [...] Second [...] Third [...]

nit: `Finally` doesn't really fit into that structure.

```suggestion
      # To access the translated text along with the fully resolved
```